src/scripts/eos-analysis

```diff
@@ -590,9 +595,14 @@ class CustomLogFormatter(logging.Formatter):
         if record.levelno not in self._MAP_LEVEL_TO_COLOR:
             levelno = logging.ERROR
 
-        symbol, color = self._MAP_LEVEL_TO_COLOR[record.levelno]
-        record.levelname = colored(symbol, color, attrs=['bold'])
-        record.msg = colored(record.msg, color)
+        if levelno != logging.DEBUG:
```

Comment:
Really don't like this if here. Can't we determine this based on the contents of colour? Maybe make it ``None`` if we don't want to change the front colour?

Reply:
I couldn't think of a nice way to do this - the problem was the the `colored` function doesn't accept anything like `None ` as an argument, as far as I can see the only way to make it print in the default colour is just to call it with two arguments not three. 
I suppose you could do:
```python
symbol, color = self._MAP_LEVEL_TO_COLOR[record.levelno]
if color:
    record.levelname = colored(symbol, color, attrs=['bold'])
    record.msg = colored(record.msg, color)
else:
    record.levelname = colored(symbol, attrs=['bold'])
    record.msg = colored(record.msg)
 ```
 Do you prefer that?

Reply:
You know what, it looks like `colored` does accept `None`, maybe there was some other issue. Let me double check this...

Reply:
Seems to work fine with `None`, so I don't know why I didn't do that before. Now changed